app/serializers/group_serializer.rb

```diff
@@ -73,6 +73,8 @@ class GroupSerializer < ApplicationSerializer
       modification_properties
     end
 
+    property :allowed_roles, item.model.class.roles.map(&:label)
```

Comment:
Ich gehe schwer davon aus dass die Mitte das mehrsprachig braucht. Verweisen wir sie dazu darauf, denselben API Endpoint einmal in jeder Sprache abzurufen?

Reply:
Ich habe mal wie besprochen einen sprachunabhängigen, vergleichbaren key hinzugefügt. Damit können sie die Rollen in mehreren Sprachen abfragen und trotzdem referenzieren.